### ISSUE TITLE
ansible-galaxy collection missing latest tag

### ISSUE THREAD

**Issue by briantward** [freeipa/ansible-freeipa#183]
Seems I cannot run the command in the README doc:

```
$ ansible-galaxy collection install freeipa.ansible_freeipa
Process install dependency map
ERROR! Cannot meet requirement * for dependency freeipa.ansible_freeipa from source 'https://galaxy.ansible.com/api/'. Available versions before last requirement added: 
Requirements from:
	base - 'freeipa.ansible_freeipa:*'
```

After some poking around at the API, I found a version 0.1.8-1 was available but was not "tagged" with latest.  Is it tag? I don't really know much about ansible galaxy terms.

https://galaxy.ansible.com/api/v2/collections/freeipa/ansible_freeipa/versions/0.1.8-1/

This worked for me as a workaround:

```
$ ansible-galaxy collection install freeipa.ansible_freeipa:0.1.8-1
Process install dependency map
Starting collection install process
Installing 'freeipa.ansible_freeipa:0.1.8-1' to '/home/bward/.ansible/collections/ansible_collections/freeipa/ansible_freeipa'
```

**Comment by varadinov** [comment]
I also faced the same issue.
@briantward thanks for saving me few hours

**Comment by t-woerner** [comment]
What exactly is missing here?

**Comment by briantward** [comment]
Apparently a duplicate of issue #180 with root cause open in Ansible https://github.com/ansible/ansible/issues/64905

Ansible collections uses SemVer, which apparently only uses hyphens to delineate special prereleases.

Of course, you could modify your versioning schema so that it doesn't contain hyphens, right? :-)

**Comment by rjeffman** [comment]
A fix for ansible/ansible#64905 has been merged a few days ago.

**Comment by t-woerner** [comment]
This has been fixed with version 0.1.10. The ansible collection on galaxy is now only using the version scheme A.B.C. But this results in not having bug fixes in galaxy without a new ansible-freeipa upstream release.
